python/ccxt/ftx.py

```diff
@@ -327,7 +327,7 @@ def describe(self):
                     'identity verification': PermissionDenied,
                     'Identity Verification': PermissionDenied,
                     'complete KYC': PermissionDenied,
-                    'you cannot trade futures': PermissionDenied
```

Comment:
Make sure to use also here our branch format(same on Jira - same branch names on different repositories).